test_api.py

```diff
@@ -62,6 +62,12 @@ def construct_parser():
     parser_payment_create.add_argument("recipient", metavar="RECIPIENT", type=str, help="the recipient of the payment")
     parser_payment_create.add_argument("amount", metavar="AMOUNT", type=int, help="the payment amount (integer, cents)")
     parser_payment_create.add_argument("message", metavar="MESSAGE", type=str, help="the message for the recipient")
+
+    parser_user_update_email = subparsers.add_parser("user_update_email", help="Update user email")
+    parser_user_update_email.add_argument("api_key_token", metavar="API_KEY_TOKEN", type=str, help="the API KEY token")
+    parser_user_update_email.add_argument("api_key_secret", metavar="API_KEY_SECRET", type=str, help="the API KEY secret")
+    parser_user_update_email.add_argument("token", metavar="TOKEN", type=str, help="the unique transaction token")
```

Comment:
this token param is not used/needed?

Reply:
the token is used to look up the User.token

Reply:
the api does not use any 'token' parameter

Reply:
changed.